public/js/user.js

```diff
@@ -135,7 +135,7 @@ function newSite(myPosition) {
     id: sites.length,
     uuid: null,
     contact: userName,
-    uid: currentUser,
+    created_by: currentUser,
```

Comment:
This line should be removed. The client does not decide the value of `created_by`.

Reply:
This function is called only when someone clicks on the 'Add new site' button. It also simplifies code, so that we don't need a `site.created_by == null` check to display edit/delete buttons. What do you think?